src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,105 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int SHIFT_BITS = 16;
+    private Node<K, V>[] table;
+    private int size;
+    private int capacity;
+    private final float loadFactor;
+    private int threshold;
+
+    public MyHashMap() {
+        this.capacity = 16;
+        this.table = (Node<K, V>[]) new Node[capacity];
+        this.loadFactor = 0.75f;
```

Comment:
The load factor '0.75f' should also be defined as a constant to improve code readability and avoid magic numbers.